nevergrad/benchmark/test_experiments.py

```diff
@@ -32,6 +32,10 @@ def test_experiments_registry(name: str, maker: tp.Callable[[], tp.Iterator[expe
     if "_pgan" in name and os.environ.get("CIRCLECI", False):
         raise SkipTest("Too slow in CircleCI")
 
+    # mixsimulator is not accepted by circleci pytest.
+    if "mixsimulator" in name and os.environ.get("CIRCLECI", False):
+        raise SkipTest("Too slow in CircleCI")
```

Comment:
it's not too slow, sigkill more likely means it takes too much memory, is that expected?

Reply:
Maybe because we are now optimizing 87600 variables (?) In the previous xp, we simulated a week of use of 4 power plants.
This time we tried to simulate an entire year of use of 10 power plants.

Reply:
Besides, we keep track of these 87600 variables every 20 budgets until all the 1600 budgets are spent.
We'll try expand this interval to 50.. It will make plots more readable by the way